app/uk/gov/hmrc/gform/LookupLoader.scala

```diff
@@ -69,23 +69,23 @@ class LookupLoader {
   private def mkRadioLookup(m: LocalisedLookupOptions): RadioLookup = RadioLookup(m)
 
   // format: off
-  private val cashType                 = read("BCD-CashType.csv",                 "ID",           "en",   "cy",   mkRadioLookup)
```

Comment:
It would be nice to keep column alignment in this block of code. So instead of all these whitespace changes only indent `currency` and `country` by two spaces.

Reply:
![Screenshot from 2020-01-22 15-31-09](https://user-images.githubusercontent.com/48124229/72907740-5577ee80-3d2c-11ea-84ce-f3b2290bfdad.png)

Like that, so it all lines up?

Reply:
Yes 👍

Reply:
Fixed in latest commit :)